packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/batchDeleteExistingOverlayActionsCommand.test.ts

```diff
@@ -5,14 +5,14 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {
-  RequestService,
-  RestHttpMethodEnum
-} from '@salesforce/salesforcedx-apex-replay-debugger/node_modules/@salesforce/salesforcedx-utils-vscode/out/src/requestService';
 import {
   INVALID_CROSS_REFERENCE_KEY,
   OVERLAY_ACTION_DELETE_URL
 } from '@salesforce/salesforcedx-apex-replay-debugger/out/src/constants';
+import {
```

Comment:
same question as above - can the import on line 11 be changed to @salesforce/salesforcedx-apex-replay-debugger?

Reply:
👍 will be covered in same follow up ticket.